Project1/Calculator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project1
+{
+    internal class Calculator
+    {
+        public int Add(int value1, int value2)
```

Comment:
Uzył bym expression body method zamiast tego(lambda =>) to co robilismy

Reply:
Nazwy dla metod zwracających wartość liczbową powinny zaczynać sie od słówka Get. Np GetSum czyli coś w postaci WeźSumę itd

Reply:
zastosuj te rzeczy do każdej z metod